### ISSUE TITLE
i2pd service fails to start

### ISSUE THREAD

**Issue by zetok** [NixOS/nixpkgs#26177]
## Issue description
i2pd service fails to start

### Steps to reproduce
1. in configuration.nix:
    
    ```nix
    services.i2pd.enable = true;
    services.i2pd.port = 12345;
    ```
2. `nixos-rebuild switch`

## Technical details

* System: 17.09pre107928.65477dcd34 (Hummingbird)
* Nix version: nix-env (Nix) 1.11.9
* Nixpkgs version: "17.09pre107928.65477dcd34"
* Sandboxing enabled: build-use-sandbox = true

error:
```
i2pd[2520]: /nix/store/ljcxsi7x7046lilv550nnjd8gmnn24vx-i2pd/bin/i2pd: line 5:  2521 Illegal instruction     /nix/store/a0gqkfyzz67lkl8w1d8wc5g59mrvlaf4-i2pd-2.13.0/bin/i2pd --conf=/nix/store/p3j7vwcxik72i3ri8zz3iv52i6pz4l57-i2pd.conf --tunconf=/nix/store/rydg2a8gkh5ll12qdma0dc50f5552p0q-i2pd-tunnels.conf
```

5th line from the `/nix/store/p3j7vwcxik72i3ri8zz3iv52i6pz4l57-i2pd.conf`:

```
netid = 2
```

**Comment by joachifm** [comment]
cc @edwtjo

**Comment by edwtjo** [comment]
@zetok what is your output of `grep avx -c /proc/cpuinfo`, should be fixed with 1e51fdc2cd77f3f68fe12406de002dd21bdcae01

**Comment by zetok** [comment]
@edwtjo 
```
grep avx -c /proc/cpuinfo
0
```

It's the QEMU's `-cpu kvm64`
